contrib/devtools/lint-extern-fundecl.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#
+# Check unnecessary extern keyword usage for function declarations.
+
+REGEXP_EXCLUDE_FILES_WITH_PREFIX="src/(crypto/ctaes/|leveldb/|secp256k1/|tinyformat.h|univalue/)"
```

Comment:
Seems not specific to extern script. It would be nice in a followup to have move common variables and functions to a file that could be shared by different linters.

Reply:
Good point. I can fix that once this PR is merged :-)